platooning_strategic/src/platoon_strategic.cpp

```diff
@@ -842,7 +842,12 @@ namespace platoon_strategic
                 status_msg.leader_cmd_speed = platoon_leader.commandSpeed;
                 status_msg.host_platoon_position = pm_.getNumberOfVehicleInFront();
 
-                status_msg.desired_gap = std::max(config_.standStillHeadway, config_.timeHeadway *  current_speed_);
+                int numOfVehiclesGaps = pm_.getNumberOfVehicleInFront() - pm_.dynamic_leader_index_;
```

Comment:
Just to make sure I understand the math, this `dynamic_leader_index_` is a 0-based index right?